packages/plans-grid-next/src/_media-queries.scss

```diff
@@ -5,35 +5,19 @@ $plan-features-header-banner-height: 20px;
 
 // Plan Features Grid Breakpoints
 $plans-2023-small-breakpoint: 780px;
-$plans-2023-medium-breakpoint: 1350px;
 
 $minWidthToGridWidthMap: (
 	780px: "668px",
 	1024px: "860px",
 	1200px: "100%",
 );
 
-// Plan Comparison Grid Breakpoints
-$plans-comparison-small-breakpoint: 880px;
-$plans-comparison-medium-breakpoint: 1024px;
-$plans-comparison-large-breakpoint: 1281px;
-
-$planComparisonMinWidthToGridWidthMap: (
-	780px: "686px",
-	880px: "870px",
-	1024px: "1024px",
-	1281px: "1238px",
-);
-
-// TODO: Replace with use of $plan-features-sidebar-width
-$planComparisonMinWidthToGridWidthWithSidebarMap: (
-	1052px: "686px",
-	1152px: "870px",
-	1296px: "1024px",
-	1553px: "1238px",
-);
-
-
+/**
+ * @deprecated
+ * TODO clk used in plans-features-main / should migrate there
```

Comment:
What does 'clk' refer to?

Reply:
my initials :-)

Reply:
p.s. I try to only add `TODO`s in code for things that I know will be worked on next. Otherwise, these lingering in the code can soon lose context